conjure-java-core/src/integrationInput/java/com/palantir/product/CovariantListExample.java

```diff
@@ -65,8 +65,8 @@ public String toString() {
         return "CovariantListExample{items: " + items + ", externalItems: " + externalItems + '}';
     }
 
-    public static CovariantListExample of(List<Object> items, List<ExampleExternalReference> externalItems) {
-        return builder().items(items).externalItems(externalItems).build();
+    public static CovariantListExample of(List<ExampleExternalReference> externalItems, List<Object> items) {
```

Comment:
Is it ok to change this order?

Reply:
This order shouldn't change, good catch.

Reply:
Addressed in https://github.com/palantir/conjure-java/pull/1243/commits/15faf55f196ceb1daf6f3051529b7e8bdaab05b6